pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -40,10 +40,15 @@ func Destroy(dir string) (err error) {
 	logrus.Infof("Using Terraform to destroy bootstrap resources...")
 	err = terraform.Destroy(tempDir, platform, "-target=module.bootstrap")
 	if err != nil {
-		err = errors.Wrap(err, "failed to run terraform")
+		return errors.Wrap(err, "failed to run terraform")
 	}
 
-	return os.Rename(filepath.Join(dir, terraform.StateFileName), filepath.Join(tempDir, terraform.StateFileName))
+	tempPath := filepath.Join(dir, terraform.StateFileName+".new")
```

Comment:
Can we rename `tempPath` to something more descriptive and more distinguishable from `tempDir`?

Reply:
`preRenamePath`?

Reply:
I could live with that. I was thinking something like `tempStateFilePath`.

Reply:
> I was thinking something like `tempStateFilePath`.

Done with 795bbba -> db3b019.